Assets/Scripts/PositionTracker.cs

```diff
@@ -2,6 +2,7 @@
 
 // Keeps track of the position of the racer on which this tracker is.
 [RequireComponent(typeof(ScoreData))]
+[RequireComponent(typeof(Rigidbody))]
```

Comment:
fyi you can also do [RequireComponent(typeof(Type1), typeof(Type2))]

otherwise good job